multimedia/tvheadend/Makefile

```diff
@@ -1,14 +1,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tvheadend
-PKG_VERSION:=2022-11-20
-PKG_RELEASE:=3
-
-PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://github.com/tvheadend/tvheadend.git
-PKG_MIRROR_HASH:=9c1bb3eea3f3539454d17e4c6aabc774a104a772aa34ed39f248521cf6488ce5
-PKG_SOURCE_VERSION:=0ff96106aa2e0f9a384c3a2662ca005797a6b399
-PKG_SOURCE_DATE:=2022-11-20
+PKG_VERSION:=9df7d2d6bc37b8aa25ac63be7b0a5d69be10c892
+PKG_RELEASE:=4
+
+PKG_SOURCE_URL:=https://github.com/tvheadend/tvheadend/archive/
+PKG_SOURCE:=$(PKG_VERSION).zip
```

Comment:
No, the tarball is going to be downloaded as 9df7d2d6bc37b8aa25ac63be7b0a5d69be10c892.zip that's not wanted. Keep the previous solution.

Reply:
I tried that twice. The hash differs from my system. I don't know how to fix it.